packages/shorebird_cli/lib/src/commands/run_command.dart

```diff
@@ -83,4 +93,20 @@ class RunCommand extends ShorebirdCommand
 
     return flutter.exitCode;
   }
+
+  /// Creates a list that is the union of [baseValidators] and
+  /// [_runValidators].
+  List<Validator> _allValidators({
```

Comment:
Can we avoid duplicating this code? I think it'd be nice to refactor `validators` to be a `Set` and each command can add it's own custom validators to the default set without having to compute the union like this. Thoughts?

Reply:
Yaa, making it a `Set` will solve this. Also we have validators on command level only. So, I think we should do this change.

Reply:
I'll update it accordingly

Reply:
@felangel Have updated to Set. but `ensure_build` test is failing. Can you check that ?

Reply:
Did you re-run `build_runner`?

Reply:
Yes, I did.